Below is an example of handling ValueError exception "try" and "except".

Here, the user is required to provide a valid integer values for calculating cylinder dimensions.

`
pwd
`{{execute}}

List the file in the current working directory using ls command.

`
ls -l volume-surface-area3.py
`{{execute}}

View the contents of the scripts file.

`
cat volume-surface-area3.py
`{{execute}}

Try to enter characters for radius and height. User is again prompted to enter valid Integer values.

Run the script below to check how ValueError is handled.

`
python volume-surface-area3.py
`{{execute}}

